string deletion_cmd
---
string deletion_cmd_response